demo/views/demo-all.dust

```diff
@@ -46,14 +46,11 @@
   <input type="text" class="offleft">
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
```

Comment:
@marijohannessen I quickly ran `npm run dev` using demo.js instead of carbon-components.min.js and it seems to be working. Do you still have the issues here?